README.md

```diff
@@ -127,58 +128,53 @@ It's possible to package the repo's example application with `electron-builder`.
-```
-
-### Test coverage
+### Open the Trace Viewer
+To open the **Trace Viewer**, select the **Trace Viewer** icon in the left sidebar:
 
-The following command prints a coverage report to the terminal. As of now it covers all typescript files of the project, including those that are not supposed to have tests.
+![Trace Viewer Icon](https://raw.githubusercontent.com/ebugden/theia-trace-extension/update-readme/doc/images/theia-trace-extension-icon.png)
 
-```shell
-yarn test --coverage --collectCoverageFrom='src/**/*.ts'
-```
+#### Add Trace Viewer to Sidebar
+If the **Trace Viewer** icon is not in the left sidebar, select menu **View** from the top-level menu and then select **Trace Viewer** in the list of views. The **Trace Viewer** icon will appear on the left, below the **File Explorer** Icon.
 
-## Using the trace extension
-This section describes how to operate the Theia trace extension to view and analyze traces. The UI, view interactions, and UX are prelimiary and subject to revisions in the future.
+![Add Trace Viewer](https://raw.githubusercontent.com/ebugden/theia-trace-extension/update-readme/doc/images/theia-trace-extension-add-to-sidebar.png)
 
-![theia-trace-extension](https://raw.githubusercontent.com/theia-ide/theia-trace-extension/master/doc/images/theia-trace-extension-0.0.2.png)
+![Trace Viewer in Sidebar](https://raw.githubusercontent.com/ebugden/theia-trace-extension/update-readme/doc/images/theia-trace-extension-sidebar.png)
 
 ### Open a trace
-To open a trace in the Theia Trace Extension, use the **File Explorer** in Theia to navigate to the trace directory. Then right-mouse click on the trace and select menu **Open With->Open Trace**.
-
-![Open Trace](https://raw.githubusercontent.com/theia-ide/theia-trace-extension/master/doc/images/theia-trace-extension-open-traces-0.0.2.png)
+When opening a trace, if the selection is a single trace file, then the tool will just open the file selected.
 
-If the selection is a single file, then the tool will open the file directly as a trace.
+The example Trace Compass trace server above supports LTTng Kernel and UST traces. Example LTTng traces can be retrieved from the [Trace Compass Tutorials](https://github.com/dorsal-lab/tracevizlab). Just download the archive [TraceCompassTutorialTraces](https://github.com/dorsal-lab/tracevizlab/blob/master/labs/TraceCompassTutorialTraces.tgz), extract them into a local directory on your computer. They can also be automatically downloaded by running `yarn download:sample-traces` from the repository's root.
 
+#### Create a trace group
 If the selection is a directory, then the tool will look for traces in **Common Trace Format (CTF)** format, such as **Linux Tracing Toolkit traces (LTTng)** Kernel and UST (Userspace) traces, and open all found CTF traces together under the same timeline. The trace events of each CTF trace will be analyzed in chronological order. With this you're able to open e.g. LTTng Kernel and UST Traces at the same time.
 
```

Comment:
After a trace was opened you should describe the different views shown, i.e. `Opened Traces`, `Available Views` and `Item properties`. Then describe how to add different a view by clicking on an available view.

Reply:
- [ ] Describing each section: I don't think we need to describe what will appear after they open a trace. I don't have strong feelings against adding this content though so let me know if it's still important and I'll add it.
- [x] How to open views: I added a section